src/components/general.js

```diff
@@ -1,12 +1,13 @@
-import {StyleSheet} from 'react-native';
+import {StyleSheet} from 'react-native';;
```

Comment:
```suggestion
import {StyleSheet} from 'react-native';
```

Aqui tem dois ponto e vígula mesmo?